Docs/README-csharp-v2.md

```diff
@@ -0,0 +1,255 @@
+# Usage examples - C&#35;
+
+## Basics
+
+The PhaxioClient class is the entry point for any Phaxio operation (you'll also need to add ```using Phaxio.V2```).
+
+    var phaxio = new PhaxioV2Client(key, secret);
```

Comment:
Another option: We could change the object name to PhaxioApiClient but provide a factory method off of the old client object, so ```var phaxio = Phaxio.V2(key, secret)```